src/python/bot/tasks/fuzz_task.py

```diff
@@ -1904,12 +1904,23 @@ def run(self):
           'Unable to setup data bundle %s.' % self.fuzzer.data_bundle_name)
       return
 
+    # For some binaries, we specify trials, which are sets of flags that we only
+    # apply some of the time. Adjust APP_ARGS for them if needed.
+    trials.setup_additional_args_for_app()
```

Comment:
What is this for?

Reply:
This was incorrectly included when resolving a conflict. Removed.